.env

```diff
@@ -15,3 +15,11 @@ SIGNATORY="0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6"
 POLYGONSCAN_API_KEY="HYGZ2HVGXGIBK6NFWWGFBIP8BEE4JWV84G"
 
 GOVERNANCE_TOKEN=0x5FbDB2315678afecb367f032d93F642f64180aa3
+
+GAME_OWNER_ADDRESS=0x70997970C51812dc3A010C7d01b50e0d17dc79C8
```

Comment:
Game Owner should probably not be relevant for Governance, which functions is it needed?

Reply:
Actually i just figured out that we only use Game owner address for GameToken constructor. We dont need GameToken at all as the only functionality is just setting the ownership address, so it means we can refactor it like we did on LandToken project.

@moritz-mg